src/components/TagWall/_mixins.scss

```diff
@@ -16,6 +16,7 @@
   > #{$tag--interactive__namespace} {
     margin-right: $carbon--spacing-03;
     max-width: 40ex;
+    justify-content: space-between;
```

Comment:
If a tag is made wider than its content (like if a width is set to 100% or just a larger pixel value), then the tag content + the close button are both centered inside the tag, which can look strange. This change will just ensure that the text is on the left, and the close button is on the right. You can see what I mean if you look at the `dev` branch deploy and set a specific width on a tag, like 300px or something larger than the natural width.